modules/manage/partials/audit-logging.adoc

```diff
@@ -129,16 +133,67 @@ a|* DescribeAcls
 * ListTransactions
 
 |heartbeat
-a|* Heartbeat
+a|
+* Heartbeat
 
 |authenticate
-a|* All authentication events
+a|
+* All authentication events
 
 |schema_registry
-a|* All Schema Registry API calls
+a|
+* All Schema Registry API calls
 
 |admin
-a|* All Admin API calls
+a|
+* All Admin API calls
+|===
+
+=== Logged events
```

Comment:
I can't see these rendered on the page.

Reply:
@BenPope ,
This material was inadvertently included in the PR--I've removed it, as it pertained to other updates.